Sources/Ink/Internal/Reader.swift

```diff
@@ -87,6 +89,15 @@ extension Reader {
 
         return string[startIndex..<currentIndex]
     }
+    
+    @discardableResult
+    mutating func readCharacter(in set: Set<Character>) throws -> Character {
```

Comment:
Could this be achieved with `character.isAny(of: set)`?

See `Hashable+AnyOf.swift`

Reply:
I can swap it around to use `character.isAny(of:set)` instead of `set.contains(character)` and it works–is that what you mean?